src/common/eeprom.cpp

```diff
@@ -903,9 +903,9 @@ bool sheet_is_calibrated(uint32_t index) {
 #endif
 }
 
-bool sheet_select(uint32_t index) {
+bool sheet_profile_select(uint32_t index) {
 #if (EEPROM_FEATURES & EEPROM_FEATURE_SHEETS)
-    if (index >= MAX_SHEETS || !sheet_is_calibrated(index))
+    if (index >= MAX_SHEETS)
```

Comment:
This change is not stated in commit message.

Reply:
msg added